lib/config.js

```diff
@@ -19,7 +19,8 @@ const shortHelpMessage = `Usage:
   -o, --output [APP.IPA]        Path to the output IPA filename
   -O, --osversion 9.0           Force specific OSVersion if any in Info.plist
   -w, --without-watchapp        Remove the WatchApp from the IPA before resigning
-  -W, --without-xctests         Remove the XCTests from the resigned IPA
+  -W, --without-plugins         Remove plugins (excluding XCTests) from the resigned IPA
```

Comment:
-p is available, it probably makes sense to use -p in here

Reply:
Per offline conversation moved `--parallel` to `-P` to accommodate